### PR TITLE
[Iteration 1] v4.0-beta

### PR DESCRIPTION
**Pull request by Jeckerson**
First iteration to just make possible to run application with latest Phalcon v4-beta

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,10 +1,4 @@
-# Please do not use this ignore file to define platform specific files.
-#
-# For these purposes create a global .gitignore file, which is a list of rules
-# for ignoring files in every Git repository on your computer.
-#
-# https://help.github.com/articles/ignoring-files/#create-a-global-gitignore
-
+.idea
```
Comment (sergeyklay) [inline .gitignore]: Please do not use this ignore file to define platform specific files.
https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

Comment (Jeckerson) [reply .gitignore]: 9d1d1fedfa6fcff48cddca97c25e60e93c195518 Fixed

File: composer.json [hunk 2]
```diff
@@ -11,23 +11,24 @@
     "type": "library",
     "license": "BSD-3",
     "homepage": "https://www.phalconphp.com",
-    "authors" : [
+    "authors": [
```
Comment (ruudboon) [inline composer.json]: Feel free to add yourself also

File: composer.json [hunk 3]
```diff
@@ -11,23 +11,24 @@
...
-            "email" : "mark@moderndeveloperllc.com"
+            "name": "Mark Garrett",
+            "email": "mark@moderndeveloperllc.com"
         }
     ],
     "require": {
-        "php": ">=5.5",
-        "ext-phalcon": "~3.0",
+        "php": ">=7.2",
+        "ext-phalcon": "4.0.0-beta.2",
```
Comment (ruudboon) [inline composer.json]: I'm not sure if we can track version with ext-phalcon. I think we need to use * here. cc @sergeyklay 
Maybe add ext-psr as well?

File: composer.json (hunk 2 above)
Comment (sergeyklay) [reply composer.json]: Actually, real author is Mark Garrett. I propose replace this block by
```
    "authors": [
        {
            "name": "Mark Garrett",
            "email": "mark@moderndeveloperllc.com"
        },
        {
            "name": "Contributors",
            "homepage": "https://github.com/phalcon/vokuro/graphs/contributors"
        }
    ]
```

File: composer.json (hunk 3 above)
Comment (sergeyklay) [inline composer.json]:
```suggestion
        "ext-phalcon": "~4.0.0-beta.2",
```

No need add psr. It is not a Vokuro dependency. In fact Phalcon has a lot of dependencies. Should we add all of them here?

File: composer.json [hunk 4]
```diff
@@ -11,23 +11,24 @@
...
         "swiftmailer/swiftmailer": "^5.4",
         "aws/aws-sdk-php": "~3.0"
     },
     "require-dev": {
-        "phalcon/dd": "~1.1.0"
+        "phalcon/dd": "~1.1.3",
```
Comment (sergeyklay) [inline composer.json]: Why user _need_ dd here?

File: composer.json [hunk 5]
```diff
@@ -11,23 +11,24 @@
...
+        "phalcon/ide-stubs": "4.0.0-beta.2"
```
Comment (sergeyklay) [inline composer.json]: It is just recommendation, not a dependency. I propose to remove this

File: .gitignore [hunk 6]
```diff
@@ -1,10 +1,3 @@
-# Please do not use this ignore file to define platform specific files.
-#
-# For these purposes create a global .gitignore file, which is a list of rules
-# for ignoring files in every Git repository on your computer.
-#
-# https://help.github.com/articles/ignoring-files/#create-a-global-gitignore
-
```
Comment (sergeyklay) [inline .gitignore]: Return back this comment

**Comment by sergeyklay** [comment]
@Jeckerson Can you add to each file a header like 
```
/**
 * This file is part of the Vökuró.
 *
 * (c) Phalcon Team <team@phalconphp.com>
 *
 * For the full copyright and license information, please view
 * the LICENSE file that was distributed with this source code.
 */
```

and add [LICENSE](https://github.com/phalcon/incubator/blob/master/LICENSE.txt) file

**Comment by sergeyklay** [comment]
And update README :)

image

File: composer.json (hunk 4 above)
Comment (Jeckerson) [reply composer.json]: To debug during development?

File: composer.json (hunk 5 above)
Comment (Jeckerson) [reply composer.json]: I think it make sense to have it here, as **Vokuro** is a tutorial application and normally people just learning Phalcon with it. So ide-stubs would help during first impression, IMO.

File: .gitignore (hunk 6 above)
Comment (Jeckerson) [reply .gitignore]: fe334d9

**Comment by Jeckerson** [comment]
Copyright headers will add in the next iteration, where would adapt files to PHP 7+ new syntax features.

**Pull request merged**